website/package.json

```diff
@@ -100,6 +100,7 @@
     "json-to-ast": "^2.1.0",
     "lodash.isequal": "^4.5.0",
     "luaparse": "^0.2.1",
+    "meriyah": "latest",
```

Comment:
Version should be semver-range, not just `latest`. This would prevent hazard of accidentally upgrading to a major version should the options ever change.

Reply:
changed to "^1.6.5"